### PR TITLE
Add offline invite link option to add user wizard

### PR DESCRIPTION
**Pull request by DonOmalVindula**
### Purpose
$subject

### Related Issues
- https://github.com/wso2/product-is/issues/18248
- https://github.com/wso2/product-is/issues/18685

### Related PRs
- N/A

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: .changeset/few-wombats-deny.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+"@wso2is/identity-apps-core": patch
+"@wso2is/console": patch
+"@wso2is/i18n": patch
```
Comment (NipuniBhagya) [inline .changeset/few-wombats-deny.md:4]: Let's patch my account as well.

File: identity-apps-core/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties [hunk 2]
```diff
@@ -148,7 +148,7 @@ information=Information
 user.details.submitted=User details successfully submitted
 close=Close
 other.login.options=Other login options
-sign.in.with=Sign In With
+sign.in.with=Sign in with
```
Comment (NipuniBhagya) [inline identity-apps-core/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties:151]: Let's make this consistent with other places.

Comment (DonOmalVindula) [reply identity-apps-core/apps/authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties]: Fixed with 8f324ad5fc35eed7633678a7feda80fea830554a

File: .changeset/few-wombats-deny.md (hunk 1 above)
Comment (DonOmalVindula) [reply .changeset/few-wombats-deny.md:4]: Fixed with 8f324ad5fc35eed7633678a7feda80fea830554a

**Review by brionmario — APPROVED** [review]

**Pull request merged**